code/datums/craft/recipes/furniture.dm

```diff
@@ -83,6 +83,13 @@
 		list(CRAFT_MATERIAL, 10, MATERIAL_WOOD),
 	)
 
+/datum/craft_recipe/furniture/bigcoffin
```

Comment:
```suggestion
/datum/craft_recipe/furniture/pauper
```
The actual object's path is `pauper` too so let's keep it organized?

Reply:
Good point